sql database/README.md

```diff
@@ -0,0 +1,6 @@
+### Usage
```

Comment:
Can we improve the documentation over here by providing examples and their associated outputs?
From reading the README, i'm still clueless as to which arguments are paired with which script and what are the possible capabilities of the database scripts.
Check out the BERT repo as an example: https://github.com/google-research/bert/blob/master/README.md
Specifically "Sentence (and sentence-pair) classification tasks"